src/devices/Mcp3xxx/README.md

```diff
@@ -1,14 +1,128 @@
 # MCP3xxx family of Analog to Digital Converters
 
-Note: Currently untested on the Mcp33xx family.
-
 Some devices like the Raspberry Pi cannot read analog values directly so rely on  [analog to digital converters](https://en.wikipedia.org/wiki/Analog-to-digital_converter), like the ones available from Microchip in the Mcp3000, Mcp3200 and Mcp3300 ranges. These chips can be accessed as an [SPI device](https://en.wikipedia.org/wiki/Serial_Peripheral_Interface) or manually via raw GPIO pins.
 
- You can use these converters in your project to access analog devices. The sample [Reading Analog Input from a Potentiometer](samples/README.md) demonstrates a concrete example using the Mcp3008 class.
+You can use these converters in your project to access analog devices.
 
 The following fritzing diagram illustrates one way to wire up the Mcp3008, with a Raspberry Pi and a potentiometer.
 
-![Raspberry Pi Breadboard diagram](samples/rpi-trimpot_spi.png)
+![Raspberry Pi Breadboard diagram](rpi-trimpot_spi.png)
+
+## Documentation
+
+The sample is based on following resources:
+
+- [Analog Inputs for Raspberry Pi Using the MCP3008](https://learn.adafruit.com/reading-a-analog-in-and-controlling-audio-volume-with-the-raspberry-pi) 
+- [Raspberry Pi Analog to Digital Converters](https://learn.adafruit.com/raspberry-pi-analog-to-digital-converters).
+- [Raspbery Pi Analog Input with MCP3008](https://gist.github.com/ladyada/3151375)
+- [MCP3008.py](https://github.com/adafruit/Adafruit_Python_MCP3008/blob/master/Adafruit_MCP3008/MCP3008.py)
+
+Major thanks to [Adafruit](https://adafruit.com) for providing python implementations, which were ported to C# for this sample.
+
+## Usage
+
+### Hardware elements
+
+The following elements are used in this sample:
+
+* [Potentiometer](https://www.adafruit.com/product/356)
+* [MCP3008](https://www.adafruit.com/product/856)
+
+
+### Accessing the MCP3008 via SPI
+
+The Raspberry Pi has support for SPI. You need to [enable the SPI interface on the Raspberry Pi](https://www.raspberrypi-spy.co.uk/2014/08/enabling-the-spi-interface-on-the-raspberry-pi/) since it is not enabled by default.
+
+You can use the following code to [access the MCP3008 via hardware SPI](Mcp3008.Sample.cs):
```

Comment:
wrong URL, should be samples/Mcp3008.Sample.cs

Note: double check the file name, links are case sensitive in theory